src/context/HicetnuncContext.js

```diff
@@ -22,8 +22,6 @@ export default class HicetnuncContextProvider extends Component {
 
       address: '',
 
-      op: undefined,
```

Comment:
this is duplicated. I find that alphabetizing object keys adds a nice way to make sure things are doubled and also helps code collaborators have fewer merge conflicts when adding keys :)